src/helper/VehicleHelper.js

```diff
@@ -79,6 +79,18 @@ class VehicleHelper {
         } : void 0;
     }
 
+    setOfferDropoffTime(offer) {
```

Comment:
The name of the method is not good - it is not really an "offer" - and this method does more.
Also the name of the parameter is not good.